spec/controllers/divisions_controller_spec.rb

```diff
@@ -1,5 +1,76 @@
 require 'spec_helper'
 
 describe DivisionsController, :type => :controller do
+  describe "#index" do
+    before :each do
+      Division.delete_all
 
+      @division = Division.create(id: 5, name: "hey jude", date: Date.new(2016,12,25),
```

Comment:
Awesome, I didn' t know about `let!`. I tried create divisions like you suggested, but this factories creates way too many divisions. I will try again now, but I am afraid I will still need to use `Divisions.create'`